src/pages/workspace/expensifyCard/WorkspaceExpensifyCardDetailsPage.tsx

```diff
@@ -94,6 +95,10 @@ function WorkspaceExpensifyCardDetailsPage({route}: WorkspaceExpensifyCardDetail
         });
     };
 
+    if (card?.isLoading) {
```

Comment:
We don’t need to show the loading state if the card detail has already been loaded

https://github.com/user-attachments/assets/85322e15-4d72-4a58-9eb1-01852531b488

```suggestion
    if (card?.isLoading && !card?.cardID) {
```